x-pack/plugins/integration_assistant/public/components/create_integration/create_integration_context.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { type PropsWithChildren } from 'react';
+import { KibanaContextProvider } from '@kbn/kibana-react-plugin/public';
+import { AssistantProvider } from '@kbn/elastic-assistant';
+import type { CreateIntegrationServices } from './types';
+
+interface CreateIntegrationProps {
+  services: CreateIntegrationServices;
+}
+
+export const CreateIntegrationContext = React.memo<PropsWithChildren<CreateIntegrationProps>>(
+  ({ children, services }) => {
+    const { http, docLinks, triggersActionsUi, notifications } = services;
+    const basePath = http.basePath.get();
+
+    // TODO: Implement assistant availability
+    const assistantAvailability = {
+      isAssistantEnabled: true,
+      hasAssistantPrivilege: true,
+      hasConnectorsAllPrivilege: true,
+      hasConnectorsReadPrivilege: true,
+      hasUpdateAIAssistantAnonymization: true,
+    };
+
+    // TODO: Implement telemetry
```

Comment:
Can we create some followup issues for TODOs

Reply:
They are listed in the main issue